Dockerfile.release

```diff
@@ -1,31 +1,21 @@
-FROM debian:bookworm-slim
+FROM gcr.io/distroless/cc-debian12:debug AS builder
 
-ARG TARGETPLATFORM
-
-RUN apt-get update \
-  && apt-get install -y ca-certificates gosu curl --no-install-recommends \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/*
+RUN ["/busybox/busybox", "mkdir", "/work", "/etc/relay"]
 
-ENV \
-  RELAY_UID=10001 \
-  RELAY_GID=10001
```

Comment:
I think you're going to want to keep these. It looks like the k8s manifest depends on the specific UID/GID values.

https://github.com/getsentry/ops/blob/af385f13a141f608487665407e34f180ecb10919/k8s/services/relay-pop/deployment.yaml#L166-L167

I'm not sure what `nonroot:nonroot` is going to map to in distroless. Either way, I would assume it is not guaranteed to match 1001.

Reply:
Afaik `nonroot` is `65532`, but it doesn't matter, this is just the default. The runtime can freely override this, kubernetes can choose the user, the only issue then will be possibly file permissions, but that just needs to configured correctly on the k8s side.

Since we mount the only necessary write directory with read all and write all permissions this should all work fine.